packages/server/graphql/mutations/helpers/bootstrapNewUser.ts

```diff
@@ -15,14 +16,18 @@ import createTeamAndLeader from './createTeamAndLeader'
 
 // no waiting necessary, it's just analytics
 const handleSegment = async (user: User, isInvited: boolean) => {
-  const {id: userId, createdAt, email, segmentId, picture, preferredName} = user
+  const {id: userId, createdAt, email, featureFlags, tier, segmentId, preferredName} = user
+  const domain = email.split('@')[1]
   segmentIo.identify({
     userId,
     traits: {
-      avatar: picture,
       createdAt,
       email,
-      name: preferredName
+      name: preferredName,
+      isActive: true,
+      featureFlags,
+      highestTier: tier,
+      isPatient0: domain ? userId === (await getPatientZeroByDomain(domain)).id : undefined
```

Comment:
+1 this is repeated in a couple of places, it'd be nice to have a helper method that'd do this